tools/license_header.py

```diff
@@ -83,7 +83,7 @@
 _LANGS = {'.cc':'*', '.h':'*', '.cu':'*', '.cuh':'*', '.py':'#',
           '.pm':'#', '.scala':'*', '.cc':'*', '.sh':'#', '.cmake':'#',
           '.java':'*', '.sh':'#', '.cpp':'*', '.hpp':'*', '.c':'*',
-          '.bat':'rem', '.pl':'#', '.m':'%', '.R':'#', '.mk':'#', '.cfg':'#'}
+          '.bat':'rem', '.pl':'#', '.m':'%', '.R':'#', '.mk':'#', '.cfg':'#', '.t':'#'}
```

Comment:
Should `.PL` also be included?

Reply:
let's improve it next time

Reply:
I can. But at the moment RAT only found a couple of files with this extension and hence I just manually added the licenses. 
I am tracking all the file types that should go into the license_header.py script soon after this release.